js/main.js

```diff
@@ -16,8 +16,8 @@ $entryForm.addEventListener('submit', function (event) {
   entryId.title = $entryForm.elements.title.value;
   entryId.notes = $entryForm.elements.notes.value;
 
-  entryId.EntryId = data.newEntryID;
-  data.newEntryId++;
+  entryId.entryId = data.nextEntryID;
+  data.nextEntryId++;
```

Comment:
Ok, and you will also need to unshift it into the array of entries of the data model in data.js

```js
data.entries.unshift(entryId);
```

honestly i'd rename this `entryId` variable to something like `entry` or `newEntry`.